src/Binding.lua

```diff
@@ -4,146 +4,156 @@ local Type = require(script.Parent.Type)
 
 local config = require(script.Parent.GlobalConfig).get()
 
---[[
-	Default mapping function used for non-mapped bindings
-]]
-local function identity(value)
-	return value
+local BindingImpl = Symbol.named("BindingImpl")
+
+local BindingInternalApi = {}
+
+local BindingPublicMethods = {}
```

Comment:
Should this be changed to `bindingPrototype` to be congruent with other classes?